<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="jenkins-agent-10-25-223-75" failures="0" tests="6" name="com.mypurecloud.sdk.v2.ApiClientRetryAsyncTest" time="60.736" errors="0" timestamp="2024-03-31T06:53:10 UTC" skipped="0">
  <testcase classname="com.mypurecloud.sdk.v2.ApiClientRetryAsyncTest" name="invokeAsyncTestWith_503" time="40.117"/>
  <system-out/>
  <testcase classname="com.mypurecloud.sdk.v2.ApiClientRetryAsyncTest" name="invokeAsyncTestWith_429" time="5.169"/>
  <system-out/>
  <testcase classname="com.mypurecloud.sdk.v2.ApiClientRetryAsyncTest" name="invokeAsyncTestWith_504_And_No_MaxRetryTime" time="0.107"/>
  <system-out/>
  <testcase classname="com.mypurecloud.sdk.v2.ApiClientRetryAsyncTest" name="invokeAsyncTestWith_429_And_No_MaxRetryTime" time="0.111"/>
  <system-out/>
  <testcase classname="com.mypurecloud.sdk.v2.ApiClientRetryAsyncTest" name="invokeAsyncTestWith_502" time="13.121"/>
  <system-out/>
  <testcase classname="com.mypurecloud.sdk.v2.ApiClientRetryAsyncTest" name="invokeAsyncTestWith_504" time="2.111"/>
  <system-out/>
</testsuite> <!-- com.mypurecloud.sdk.v2.ApiClientRetryAsyncTest -->
